Utilities/Analyzer.cc

```diff
@@ -71,84 +61,11 @@ class Analyzer{
     cout << "Output will be saved to " << ofname << endl;
     ofile->cd();
     t = new TTree("t","EventTree");
-    BookBranches();
```

Comment:
BookBranches() is the function that do t->Branch(); . It is simply a grouper that collect the t->Branch commands together.
Also BookBranches() is a virtual function, subject to replacement in the actual running code, so one can customize what branches to be saved into the intermediate tree.

Reply:
I see, you did BookBranches in the main analysis code.
You put BookBranches() right after SetOutput(), making that identical to what it is in here.

Reply:
What I thought the program was doing, was to book the branches in the Analyzer.cc anyways, then book more branches in the Validation or any other top level program. Was that not the case? Your event counting was essentially removed, since the same event can count in different ways under systematic variations in the cutflow, and that's handled by the RegionAssociations.

Reply:
Yes, BookBranches is done in Analyzer. Validation.cc redefined it in the derived class of Analyzer. And it will be called by Analyzer::Init()

Reply:
That wasn't the question though. It's whether the Analyzer.cc version of BookBranches that booked now deprecated branches did anything, regardless of the extended definition in the Validation?
I mostly wanted the extra branches that were by default booked by the Analyzer.cc version of BookBranches to go.